src/web_server.cpp

```diff
@@ -1128,6 +1145,7 @@ web_server_setup() {
   server.on("/settime$", handleSetTime);
   server.on("/reset$", handleRst);
   server.on("/restart$", handleRestart);
+  server.on("/restartevse$", handleRestartEvse);
```

Comment:
Should mirror the MQTT API, but will need a little bit of backwards compatibility, so I would make a GET request to /restart restart the gateway but posting to /restart with either `gateway` or `evse` restart the appropriate device.... That being said that doesn't feel very RESTful.... I guess the RESTful thing to do would be to POST `restart` to /evse or /gateway   or maybe this is just one of those cases that doesn't fit into that model...

Reply:
would like but if we do the /gateway /evse way, then it would have sense to move all endpoints related to /gateway on it. I guess /restart gateway|evse follows more actual structure.  Shouldn't we post it as json also ? /restart {"dev": "gateway"}

Reply:
yeah i think probably at this point sticking to /reset is the best. Using a JSON payload is also fine, but should also have parity with the MQTT API

Reply:
I've updated booth api & mqtt to /restart {"device": "gateway|evse"}

@mathieucarbou